x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -197,7 +198,8 @@ export class IngestManagerPlugin
       config$: this.config$,
       savedObjects: core.savedObjects,
       isProductionMode: this.isProductionMode,
-      serverInfo: this.serverInfo,
+      kibanaVersion: this.kibanaVersion,
+      coreSetup: this.coreSetup,
```

Comment:
do we really need to pass the entire `core` to app context? can we just pass `core.http`?

Reply:
Going to do the change